### PR TITLE
small error handle

### PR DESCRIPTION
**Pull request by neheb**
Signed-off-by: Rosen Penev <rosenp@gmail.com>

### CODE REVIEW

**Comment by whyman** [comment]
I personally dislike doing work in if conditions.

Can we do the work, assign a `ret` value and then check it?

The compiler will do the right thing, and its much clearer to read.

**Comment by neheb** [comment]
I mean, the surrounding does similar

**Comment by whyman** [comment]
Fix that too while you are there? 😄

**Comment by neheb** [comment]
I think this is what you meant.

File: src/transcoding/transcode_ext_handler.cc [hunk 1]
```diff
@@ -127,13 +127,17 @@ std::unique_ptr<IOHandler> TranscodeExternalHandler::serveContent(std::shared_pt
             strcpy(fifo_template, "mt_transcode_XXXXXX");
             location = tempName(config->getOption(CFG_SERVER_TMPDIR), fifo_template);
             log_debug("creating reader fifo: {}", location.c_str());
-            if (mkfifo(location.c_str(), O_RDWR) == -1) {
+            auto r = mkfifo(location.c_str(), O_RDWR);
+            if (r == -1) {
```
Comment (KarlStraussberger) [inline src/transcoding/transcode_ext_handler.cc:131]: I'd prefer `!= 0`. Looks more like error handling.

**Review by KarlStraussberger — CHANGES_REQUESTED** [review]

File: src/transcoding/transcode_ext_handler.cc (hunk 1 above)
Comment (neheb) [reply src/transcoding/transcode_ext_handler.cc]: OK

**Review by KarlStraussberger — APPROVED** [review]

**Pull request merged**